web/modules/custom/custom_page/custom_page.behat.inc

```diff
@@ -296,4 +307,34 @@ class CustomPageSubContext extends DrupalSubContextBase implements DrupalSubCont
     return $this->getOgMenuTree($menu_instance);
   }
 
+  /**
+   * Asserts that a custom page tile has the logo properly set.
+   *
+   * @param string $title
+   *   The document title.
+   *
+   * @throws \Exception
+   *   Thrown when the logo is not found or not correct.
+   *
+   * @Then the logo should be shown in the :title custom page tile
+   */
+  public function assertCustomPageTileHasLogo($title) {
```

Comment:
For completeness, let's put the `string` type hint for the argument and `void` for the return type.